eng/common/internal/Tools.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <TargetFramework>net472</TargetFramework>
     <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
+    <ImportDirectoryPackagesProps>false</ImportDirectoryPackagesProps>
```

Comment:
Why is this ok? Won't it get overwritten the next time arcade pushes?

Reply:
Yes it will. At the same time it is the only sane way to fix this problem. I'm patching arcade in parallel and will jsut block until we get that update. 

https://github.com/dotnet/arcade/pull/14774